admin_guide/diagnostics_tool.adoc

```diff
@@ -157,36 +168,139 @@ A failure from this check indicates that a significant amount of space in etcd
+playbook:
+
+----
+# ansible-playbook -i <inventory_file> \
+    /usr/share/ansible/openshift-ansible/playbooks/common/openshift-checks/health.yml
+----
+
+To set variables in the command line, include the `-e` flag with any desired
+variables in `key=value key=value` format. For example:
+
+----
+# ansible-playbook -i <inventory_file> \
+    /usr/share/ansible/openshift-ansible/playbooks/common/openshift-checks/health.yml \
+    -e openshift_check_logging_index_timeout_seconds=30
+    -e etcd_max_image_data_size_bytes=40000000000
+----
+
+To disable specific checks, include the variable `openshift_disable_check` with
+a comma-delimited list of check names in your inventory file before running the
+playbook. For example:
+
+----
+openshift_disable_check=etcd_traffic,etcd_volume
+----
+
+Alternatively, set any checks you want to disable as environment variables with `-e openshift_disable_check` when running the `ansible-playbook` command.
+
+[[admin-guide-health-checks-via-docker-cli]]
+=== Running Health Checks via Docker CLI
+
+It is possible a playbook may require dependencies that are not installed
+locally on the host running the `ansible-playbook` command. You can avoid this
+issue by running the *openshift-ansible* health checks directly from the
+ifdef::openshift-enterprise[]
+*ose-ansible*
+endif::[]
+ifdef::openshift-origin[]
+*origin-ansible*
+endif::[]
+image via the Docker CLI.
+
+To do so, specify your cluster's inventory file and the *_health.yml_* playbook
+when running the following `docker run` command as a non-root user that has
+privileges to run containers:
+
+----
+# docker run -u `id -u` \ <1>
+    -v $HOME/.ssh/id_rsa:/opt/app-root/src/.ssh/id_rsa:Z,ro \ <2>
+    -v /etc/ansible/hosts:/tmp/inventory:ro \ <3>
+    -e INVENTORY_FILE=/tmp/inventory \
+    -e PLAYBOOK_FILE=playbooks/byo/openshift-checks/health.yml \ <4>
+    -e OPTS="-v -e openshift_check_logging_index_timeout_seconds=30 -e etcd_max_image_data_size_bytes=40000000000" \ <5>
+ifdef::openshift-enterprise[]
+    openshift3/ose-ansible
+endif::[]
+ifdef::openshift-origin[]
+    openshift/origin-ansible
+endif::[]
+----
+<1> These options make the container run with the same UID as the current user,
+which is required for permissions so that the SSH key can be read inside the
+container (SSH private keys are expected to be readable only by their owner).
+<2> Mount SSH keys as a volume under *_/opt/app-root/src/.ssh_* under normal usage
+when running the container as a non-root user.
+<3> Change *_/etc/ansible/hosts_* to the location of your cluster's inventory file,
+if different. This file will be bind-mounted to *_/tmp/inventory_*, which is
+used by the `INVENTORY_FILE` environment variable in the container.
+<3> The `PLAYBOOK_FILE` environment variable is set to the location of the
+*_health.yml_* playbook inside the container.
+<4> Set any desired variables that accept user-defined values in `key=value key=value` format.
```

Comment:
All variables can be user-defined so that's kind of redundant. The example's clear enough but I'd suggest this say:

> Set any variables desired for a single run with the `-e key=value` format.

Having two in there could be confusing.